package.json

```diff
@@ -292,14 +290,21 @@
 	},
 	"resolutions": {
 		"react-dates": "17.1.1",
+		"@types/wordpress__block-editor": "npm:noop-package@1.0.0",
+		"@types/wordpress__editor": "npm:noop-package@1.0.0",
+		"@types/wordpress__notices": "npm:noop-package@1.0.0",
+		"@types/wordpress__rich-text": "npm:noop-package@1.0.0",
```

Comment:
Same reason as above. We need to specifically noop those packages since they'll also attempt to redeclare the `@wordpress/data` module.